### PR TITLE
feat(feature-toggle): add new package for feature check decorator

### LINKED ISSUE #837: package for feature-toggle
**Is your feature request related to a problem? Please describe.**
Create a new package for feature-toggle 

**Describe the solution you'd like**
Create a new package for feature-toggle

### PR DESCRIPTION
**Pull request by yeshamavani**
BREAKING CHANGE:
decorator removed from service and moved to package

gh-837

## Description

Created a new package extension for feature-toggle, that provides a method level decorator to check if feature is allowed or not.

Fixes #837 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## Checklist:

- [x] Performed a self-review of my own code
- [x] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/feature-toggle/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# @sourceloop/feature-toggle
```
Comment (akshatdubeysf) [inline packages/feature-toggle/README.md:1]: This still seems to be about the Feature Toggle Service

File: packages/core/package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
   "main": "dist/index.js",
```
[the remark this answers is not in the document]
Comment (yeshamavani) [reply packages/core/package.json]: by mistake, reverted!!

File: packages/feature-toggle/src/keys.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import {FeatureFlagFn, FeatureFlagMetadata} from './types';
+
+/**
+ * Binding keys used by this component.
+ */
+// export namespace FeatureToggleComponentBindings {
+//   export const COMPONENT = BindingKey.create<FeatureToggleComponent>(
+//     `${CoreBindings.COMPONENTS}.FeatureToggleComponent`,
+//   );
+// }
```
Comment (akshatdubeysf) [inline packages/feature-toggle/src/keys.ts:11]: ??

**Review by akshatdubeysf — CHANGES_REQUESTED** [review]

File: packages/feature-toggle/src/keys.ts (hunk 3 above)
Comment (yeshamavani) [reply packages/feature-toggle/src/keys.ts]: removed

File: packages/feature-toggle/src/__tests__/unit/README.md [hunk 4]
```diff
@@ -0,0 +1 @@
+# Unit tests
```
[the remark this answers is not in the document]
Comment (yeshamavani) [reply packages/feature-toggle/src/__tests__/unit/README.md:1]: added.

**Review by samarpan-b — CHANGES_REQUESTED** [review]
should be generated using lb4 extension

**Pull request merged**